valhalla/baldr/traffictile.h

```diff
@@ -67,12 +67,16 @@ struct TrafficSpeed {
         assert(false);
     }
   }
-  /// Returns overall speed in kph across edge
+  // Returns overall speed in kph across edge
   inline uint8_t get_overall_speed() const volatile {
     return overall_speed << 1;
   }
 
-  /// Returns speed in a certain subsegment in kph
+  // Returns speed in a certain subsegment in kph
+  //
+  // CAUTION: Caller must check that returned value is less or equal to
+  // `baldr::MAX_TRAFFIC_SPEED_KPH`. A larger value means the live-speed
+  // is not known
```

Comment:
i made some related changes to this in #2451 can you let me know if i should revert them in favor of this?